### PR TITLE
Added types for actions

### PR DESCRIPTION
**Pull request by TGMM**
This PR:
- Adds the actions.ts file, where it adds a type for every possible action
- Adds an action handlers dictionary to lookup the sent action
- Adds a function that serializes actions
- Adds a config file for the biome formatter

### CODE REVIEW

**Comment by TGMM** [comment]
Before I merge this, be sure to let me know what your preferred formatting options are. Such as: double quotes vs single quotes, spaces vs tabs, semi colon at the end, trailing comma, etc. I'm fine with whatever you choose, I just want it to be consistent.

File: Server/src/main.ts [hunk 1]
```diff
@@ -1,110 +1,94 @@
...
-            lobbyInfoAction(client)
-            break
-        }
+        const message: ActionClientToServer = stringToJson(msg);
+        const { action, ...actionArgs } = message;
+
+        // This only works for now, once we add more arguments
+        // we'll need to refactor this
+        // Maybe add a context type that includes everything
+        // connection related?
```
Comment (V-rtualized) [inline Server/src/main.ts:52]: A single context object for action, arguments, and client/socket?

**Review by V-rtualized — APPROVED** [review]
Looks great!

**Comment by V-rtualized** [comment]
> Before I merge this, be sure to let me know what your preferred formatting options are. Such as: double quotes vs single quotes, spaces vs tabs, semi colon at the end, trailing comma, etc. I'm fine with whatever you choose, I just want it to be consistent.

I prefer single quotes, tabs, no semi colon, otherwise anything works. BiomeJS looks great, haven't heard of it before.

**Comment by TGMM** [comment]
> I prefer single quotes, tabs, no semi colon, otherwise anything works. BiomeJS looks great, haven't heard of it before.
I'll merge this and create a new PR for formatting changes

**Pull request merged**